### DISCUSSION TITLE
Why am I unable to install from PyPI using pip ?

### DISCUSSION THREAD

**Discussion by Adeel-Intizar** [aahnik/wappdriver#48] [Q&A] [answered]
Hello, Thank you for your great work
I can't install it using pypi (pip install wappdriver)

![image_2021-02-06_153220](https://user-images.githubusercontent.com/56041154/107115807-f29eba00-6890-11eb-954e-30ccf2d5ccd7.png)

**Comment by aahnik** [comment, marked as answer]
Hey! @Adeel-Intizar , I checked its **perfectly finely installable from PyPI using `pip` in all three OS : mac, windows and linux.**

Make sure you have `python 3.9` installed along with the latest version of `pip`.

Run the commands and share me the results:

1. `python --version`
2. `pip --version`
